src/ext/java/org/opentripplanner/ext/vehicleparking/bikely/BikelyUpdater.java

```diff
@@ -0,0 +1,95 @@
+package org.opentripplanner.ext.vehicleparking.bikely;
+
+import static org.opentripplanner.routing.vehicle_parking.VehicleParkingState.OPERATIONAL;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Currency;
+import org.opentripplanner.routing.vehicle_parking.VehicleParking;
+import org.opentripplanner.routing.vehicle_parking.VehicleParkingSpaces;
+import org.opentripplanner.routing.vehicle_parking.VehicleParkingState;
+import org.opentripplanner.transit.model.basic.LocalizedMoney;
+import org.opentripplanner.transit.model.basic.LocalizedString;
+import org.opentripplanner.transit.model.basic.Money;
+import org.opentripplanner.transit.model.basic.NonLocalizedString;
+import org.opentripplanner.transit.model.basic.WgsCoordinate;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.updater.GenericJsonDataSource;
+
+/**
+ * Vehicle parking updater class for the Norwegian bike box provider Bikely:
+ * https://www.safebikely.com/
+ */
+public class BikelyUpdater extends GenericJsonDataSource<VehicleParking> {
+
+  private static final String JSON_PARSE_PATH = "result";
+  private static final Currency NOK = Currency.getInstance("NOK");
+  private final String feedId;
+
+  public BikelyUpdater(BikelyUpdaterParameters parameters) {
+    super(parameters.url(), JSON_PARSE_PATH, parameters.httpHeaders());
+    this.feedId = parameters.feedId();
+  }
+
+  @Override
+  protected VehicleParking parseElement(JsonNode jsonNode) {
+    var vehicleParkId = new FeedScopedId(feedId, jsonNode.get("id").asText());
+
+    var address = jsonNode.get("address");
+    var workingHours = jsonNode.get("workingHours");
```

Comment:
Are these always open 24/7 (if not under maintenance)?

Reply:
The API has fields for the opening hours but they are always set to `null`. Presumably this will contain opening hours at some stage but right now it doesn't.